src/app/components/application/personal-info/parent/parent.component.ts

```diff
@@ -162,6 +163,7 @@ export class ParentComponent implements OnInit {
   }
 
   removeParent(parentContactId: string): void {
+    this.isLoding = true;
```

Comment:
can you fix the variable name? should be `isLoading`. You missed an 'a' in there